packages/extension/src/RichTextActions/RichTextActionsImpl.tsx

```diff
@@ -355,7 +361,15 @@ If the user provides a prompt that you cannot produce a document for, you should
 Do not converse with the user.
   - Do not ask clarifying questions.
   - Do not add conversational preamble such as \`Sure, I can do that\`, or \`Ok, here's the change\`.
-  - Do not include [DIALOG] if you've successfully produced a document.`,
+  - Do not include [DIALOG] if you've successfully produced a document.
+  
+  ${
+    keywords.length
```

Comment:
I think the same thing needs adding into the `rewriteSelectedContentUsingAI` method.